### PR TITLE
Update adding-outside-collaborators-to-repositories-in-your-organizat…

### PR DESCRIPTION
**Pull request by queenofcorgis**
…ion.md

Proposing this based on a community question: https://github.com/orgs/community/discussions/78092

### Why:  When a community member asked in the link above I expected to find this information in the docs, this change disseminates the information more widely than just a post in the community. 

Closes: 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added sentence `Outside collaborators cannot be added to a team, only members of an organization can be.`

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by myarb** [comment]
👋🏼 Hello again, @queenofcorgis ! Thanks for another PR to update GitHub Docs ✨ I'll link to your other open PR, as they're related enough to review together: https://github.com/github/docs/pull/30444. I've triaged it for review by someone on our team.

**Comment by queenofcorgis** [comment]
Thanks @myarb for linking and thanks to you and all the docs team, big fan of your work!

File: content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ If your organization requires two-factor authentication, all outside collaborato
 Outside collaborators cannot be added to an {% data variables.enterprise.prodname_emu_enterprise %}. However, you can grant limited access to users outside your enterprise using the guest collaborator role. For more information, see "[AUTOTITLE](/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/roles-in-an-enterprise#guest-collaborators)."
 {% endif %}
 
+Outside collaborators cannot be added to a team, only members of the organization can be.
```
Comment (sophietheking) [inline content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md:47]:
```suggestion
Outside collaborators cannot be added to a team, team membership is restricted to members of the organization.
```
Slight edit to the second half of the sentence.

**Review by sophietheking — DISMISSED** [review]
@queenofcorgis 👋 Thanks for opening this PR! I left a suggestion to slightly alter the wording. Please feel free to `@mention` me to get this merged.

**Comment by queenofcorgis** [comment]
@sophietheking love the suggestion and good to "run into" you 😃  I committed it!

**Review by sophietheking — APPROVED** [review]

**Pull request merged**